### PR TITLE
Implemented a page for payments use-case

### LINKED ISSUE #14217: Implement new page for payments use case
The task is to implement a brand new page to our website!

Content worked on here: https://github.com/ethereum/ethereum-org-website/issues/13653

**Instructions**
URL for the page should be:  /payments

Design and content of the page:
https://www.figma.com/design/PlJHPXisq7b5ual2DjrR9F/payments?node-id=12940-869&t=J998WMcKEFD6c0jn-1

this design follows the same template as for example: https://ethereum.org/en/defi/ or https://ethereum.org/en/nft/

if you have any questions feel free to comment in figma or in this issue!

### PR DESCRIPTION
**Pull request by MukulKolpe**
## Description

Implemented a new page for the Ethereum Payments (`/payments`) use-case.

Content worked on here: https://github.com/ethereum/ethereum-org-website/issues/13653 by @BoluwatifeInferno.

Design and content of the page:
https://www.figma.com/design/PlJHPXisq7b5ual2DjrR9F/payments?node-id=12940-869&t=J998WMcKEFD6c0jn-1

## Related Issue

Resolves #14217

### CODE REVIEW

**Comment by konopkja** [comment]
@MukulKolpe  do you think you can still look at this or should core devs take over here?

**Comment by MukulKolpe** [comment]
> I wanted to review this but preview is broken

Thanks for pointing that out. It should be fixed now. Let me know what you think!

**Comment by MukulKolpe** [comment]
> @MukulKolpe do you think you can still look at this or should core devs take over here?

For sure! Let me know if I've missed anything or if any adjustments are needed.

**Comment by konopkja** [comment]
preview: https://deploy-preview-14370--ethereumorg.netlify.app/en/payments/

**Comment by konopkja** [comment]
change request:

- [ ] max width 320px for image
- [ ] remove "ethereum in action" title and its one sentence


Screenshot 2024-12-16 at 13 44 17

File: public/content/payments/index.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+---
+title: Ethereum Payments
+description: An overview of payments on Ethereum
+lang: en
+template: use-cases
+emoji: ":frame_with_picture:"
+sidebarDepth: 2
+image: /images/impact_transparent.png
+alt: An Eth logo being displayed along with giving hands.
+summaryPoint1: A world where money moves as freely as information.
```
Comment (konopkja) [inline public/content/payments/index.md:10]:
```suggestion
summaryPoint1: A world where money moves as freely as information
summaryPoint2: Open and global, enabling borderless transactions for everyone
summaryPoint3: Payments received within a minute 


```

**Review by konopkja — CHANGES_REQUESTED** [review]

**Review by corwintines — APPROVED** [review]
Thanks @MukulKolpe

**Pull request merged**